[2025-04-03T11:09:57.986+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-03T11:09:58.039+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag_name.create_dataset manual__2025-04-03T11:09:49.495189+00:00 [queued]>
[2025-04-03T11:09:58.057+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag_name.create_dataset manual__2025-04-03T11:09:49.495189+00:00 [queued]>
[2025-04-03T11:09:58.058+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-03T11:09:58.092+0000] {taskinstance.py:2890} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_dataset> on 2025-04-03 11:09:49.495189+00:00
[2025-04-03T11:09:58.112+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2239) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-03T11:09:58.120+0000] {standard_task_runner.py:72} INFO - Started process 2244 to run task
[2025-04-03T11:09:58.130+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_dag_name', 'create_dataset', 'manual__2025-04-03T11:09:49.495189+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/intermediaire_table.py', '--cfg-path', '/tmp/tmpi8p1pftc']
[2025-04-03T11:09:58.132+0000] {standard_task_runner.py:105} INFO - Job 111: Subtask create_dataset
[2025-04-03T11:09:58.327+0000] {task_command.py:467} INFO - Running <TaskInstance: my_dag_name.create_dataset manual__2025-04-03T11:09:49.495189+00:00 [running]> on host d6badcb20287
[2025-04-03T11:09:58.554+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag_name' AIRFLOW_CTX_TASK_ID='create_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-04-03T11:09:49.495189+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-03T11:09:49.495189+00:00'
[2025-04-03T11:09:58.563+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-03T11:09:58.564+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-03T11:09:58.564+0000] {logging_mixin.py:190} INFO - Current task name:create_dataset state:running start_date:2025-04-03 11:09:58.040435+00:00
[2025-04-03T11:09:58.565+0000] {logging_mixin.py:190} INFO - Dag name:my_dag_name and current dag run status:running
[2025-04-03T11:09:58.565+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-03T11:09:58.604+0000] {base.py:84} INFO - Retrieving connection 'gcp_conn'
[2025-04-03T11:09:58.741+0000] {bigquery.py:589} INFO - datasetId was not specified in `dataset_reference`. Will use default value fake_news_detection.
[2025-04-03T11:09:58.742+0000] {bigquery.py:589} INFO - projectId was not specified in `dataset_reference`. Will use default value nlpfakenews.
[2025-04-03T11:09:58.743+0000] {bigquery.py:600} INFO - Creating dataset: fake_news_detection in project: nlpfakenews 
[2025-04-03T11:09:58.973+0000] {bigquery.py:2137} INFO - Dataset fake_news_detection already exists.
[2025-04-03T11:09:59.031+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-03T11:09:59.033+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_dag_name, task_id=create_dataset, run_id=manual__2025-04-03T11:09:49.495189+00:00, execution_date=20250403T110949, start_date=20250403T110958, end_date=20250403T110959
[2025-04-03T11:09:59.087+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-03T11:09:59.088+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-03T11:09:59.089+0000] {logging_mixin.py:190} INFO - Dag name:my_dag_name queued_at:2025-04-03 11:09:50.628713+00:00
[2025-04-03T11:09:59.089+0000] {logging_mixin.py:190} INFO - Task hostname:d6badcb20287 operator:BigQueryCreateEmptyDatasetOperator
[2025-04-03T11:09:59.122+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-03T11:09:59.193+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-03T11:09:59.196+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
